src/svg-icons/index-generator.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const rrs = require('recursive-readdir-sync');
+
+var outArray = [];
+outArray.push('module.exports = {\n');
+
+rrs('./').forEach(function(file) {
+	if(file !== 'index-generator.js' && file !== 'index.js')
+	{
+		var fileLines = fs.readFileSync(file, 'utf8').split('\n');
```

Comment:
My bad, we don't enforce indent in the `.eslintrc`. Ok, let's keep it like it. I think that once we take care of the huge number of PR, we can use the autofix of eslint to clean this mess.